src/Tribe/Customizer/Global_Elements.php

```diff
@@ -154,18 +154,61 @@ public function setup() {
 	public function register_settings( WP_Customize_Section $section, WP_Customize_Manager $manager ) {
 		$customizer = Tribe__Customizer::instance();
 
-		// Add an heading that is a Control only in name: it does not, actulally, control or save any setting.
+		// Add an heading that is a Control only in name: it does not, actually, control or save any setting.
 		$manager->add_control(
 			new Heading(
 				$manager,
-				$customizer->get_setting_name( 'link_color_heading', $section ),
+				$customizer->get_setting_name( 'font_color_heading', $section ),
 				[
 					'label'   => esc_html__( 'Set Font Colors', 'the-events-calendar' ),
 					'section' => $section->id,
 				]
 			)
 		);
 
+		$manager->add_setting(
+			$customizer->get_setting_name( 'event_title_color', $section ),
+			[
+				'default'              => '#141827',
+				'type'                 => 'option',
+				'sanitize_callback'    => 'sanitize_hex_color',
+				'sanitize_js_callback' => 'maybe_hash_hex_color',
+			]
+		);
+
+		$manager->add_control(
+			new WP_Customize_Color_Control(
+				$manager,
+				$customizer->get_setting_name( 'event_title_color', $section ),
+				[
+					'label'       => esc_html__( 'Event Title', 'the-events-calendar' ),
+					'section'     => $section->id,
```

Comment:
:warning: Array double arrow not aligned correctly; expected 1 space(s) between "'section'" and double arrow, but found 5.

<sub>hash: 4377a39130adcfcb9ae6102fc098dcdb</sub>